tests/scripts/arm32_ci_test.sh

```diff
@@ -105,7 +114,7 @@ rm -f -v ${__ROOTFS_DIR}${ARM_CHROOT_HOME_DIR}/bin/CoreFxBinDir/libcoreclrtracep
 
 chroot ${__ROOTFS_DIR} /bin/bash -x <<EOF
     cd ${ARM_CHROOT_HOME_DIR}
-    ./tests/runtest.sh --sequential\
+    env ${COMPLUS_OPTIONS} ./tests/runtest.sh --sequential\
```

Comment:
Maybe passing `--ryujit` should affect the `--testDirFile`. And maybe that file should be dependent on the build flavor (e.g., Debug/Checked/Release). Thus, RyuJIT could get a list of passing tests for Checked and a separate list for Release. If you did that, you wouldn't need to restrict the test run to only Release builds; you could enable some set of tests (that pass) for Checked as well. Thoughts?

Reply:
I will discuss this with your comment above.